examples/models/modules/layers_3d.py

```diff
@@ -0,0 +1,84 @@
+from torch import nn
```

Comment:
Let's rename this file as `blocks.py` and add an import in `modules/__init__.py`.